sphinx/domains/python.py

```diff
@@ -571,15 +571,12 @@ class PyMethod(PyObject):
         'abstractmethod': directives.flag,
         'async': directives.flag,
         'classmethod': directives.flag,
-        'property': directives.flag,
+        'property': directives.flag,  # deprecated
         'staticmethod': directives.flag,
     })
 
     def needs_arglist(self) -> bool:
-        if 'property' in self.options:
-            return False
-        else:
-            return True
+        return True
```

Comment:
Please don't remove codes for properties from PyMethod. This breaks built document. It should only emit a warning during deprecation period.

Reply:
Done!